resources/build.js

```diff
@@ -52,13 +60,14 @@ function buildPackageJSON() {
   delete packageJSON.scripts;
   delete packageJSON.devDependencies;
 
-  const { version } = packageJSON;
-  const match = /^[0-9]+\.[0-9]+\.[0-9]+-?([^.]*)/.exec(version);
-  assert(match, 'Version does not match semver spec.');
-  const tag = match[1];
-  assert(!tag || tag === 'rc', 'Only "rc" tag is supported.');
+  const { preReleaseTag } = parseSemver(packageJSON.version);
+  if (preReleaseTag != null) {
+    const [tag] = preReleaseTag.split('.');
+    assert(tag === 'rc', 'Only "rc" tag is supported.');
```

Comment:
I see the code had this check before, but I'm just curious why only `rc` is supported?

Reply:
@martijnwalraven Thanks for the review 👍 
It was added to prevent issues similar to https://github.com/graphql/graphql-js/issues/1375
These scripts are run by Travis and package automatically published so I'm trying to use "fail fast" design for them. Especially since I don't have NPM keys to do a manual fix if something goes wrong.
If we decide to use other tags in future it will be a matter of just adding them here.